app/views/posts/new.html.erb

```diff
@@ -1,5 +1,3 @@
-
-
 <%= render 'form', post: @post %>
 
 
```

Comment:
Generally it looks pretty solid but it'll be great if we could have different sizes for our code editor box and our console box. Besides, the line where run button is located is white, and does not seem to go along quite well with the overall webpage.